README.rst

```diff
@@ -61,28 +61,60 @@ Examples_ that demonstrate how to use it.
 .. _Examples: https://skypy.readthedocs.io/en/stable/examples/index.html
 
 
-Get in Touch
+Contributing
 ------------
 
-You are welcome to talk about the SkyPy package and code using our
-`Discussions Page`_. For any other questions about the project in general,
-please get in touch with the `SkyPy Co-ordinators`_.
+We love contributions! 
+SkyPy is open source,
+built on open source, and we'd love to have you hang out in our community.
 
- .. _Discussions Page: https://github.com/skypyproject/skypy/discussions
- .. _SkyPy Co-ordinators: mailto:skypy-coordinators@googlegroups.com
+How to contribute
+^^^^^^^^^^^^^^^^^
 
-Contributing
-------------
+Whether you would like to contribute to SkyPy with your own piece of code or
+helping develop a concrete feature in SkyPy:
 
-We love contributions! SkyPy is open source,
-built on open source, and we'd love to have you hang out in our community.
-For information on how to contribute see our `Contributor Guidelines`_.
+1. Read through our `Discussions Page`_ to start a new conversation and share your
+ideas or follow up an existing conversation on a particular feature.
+
+2. Following the discussions, when you have a good idea of the specifics 
+of the feature you wish to contribute, open an `Issue`_ describing the feature. 
+
+3. Then follow the `Contributor Guidelines`_ to open a `Pull Request`_ to contribute
+the code implementing the new feature.
+
+For further information on how to contribute see our `Contributor Guidelines`_.
 All communication relating to The SkyPy Project must meet the standards set out
 in the `Code of Conduct`_.
 
+.. _Issue: https://github.com/skypyproject/skypy/issues
+.. _Pull Request: https://github.com/skypyproject/skypy/pulls
 .. _Contributor Guidelines: https://skypy.readthedocs.io/en/latest/developer/contributing.html
 .. _Code of Conduct: https://skypy.readthedocs.io/en/stable/project/code_of_conduct.html
 
+Members vs External contributions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In terms of contributing, there is very little difference between SkyPy members and external contributions!
+
+1. SkyPy members go through a simple onboarding process where their contributions are previously
+defined, they have access to internal communication channels, they are involved in decission making
+processes and attend quarterly meetings. Member's contributions
+will be acknowledged in the main author list of Zenodo.
```

Comment:
Suggested change:

> 1. SkyPy _Members_ go through a simple onboarding process where their expertise and expected contributions are discussed and defined. _Members_ have access to internal communication channels, they are involved in SkyPy decision making processes and attend quarterly meetings. _Members_ are listed as a separate tier in author lists for SkyPy publications, with the classification of "Creators" in the Zenodo DoI.

Reply:
was it creators or project members?
https://zenodo.org/record/4071945#.YmZovy1Q1QI
https://zenodo.org/record/5648283#.YmZozS1Q1QI